lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
I'm confused about this change. If `degrees += this.offset;` is moved after constraining to the user range, then why wouldn't this inversion operation be constrained to the user range. Can you give me a quick example of how this falls down?